### PR TITLE
[RDV-411] [RDV-412 ] Maintain WW/WL ans MIP when switching plane

### PR DESCRIPTION
**Pull request by kushgadhvi16**
-> Added slabThickness in Presentation Interface  
-> Added Logic to get and set slabThickness when changing hanging protocol.

### CODE REVIEW

File: extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts [hunk 1]
```diff
@@ -181,6 +182,16 @@ class CornerstoneViewportService
     if (properties) viewport.setProperties(properties);
     const camera = presentations?.positionPresentation?.camera;
     if (camera) viewport.setCamera(camera);
+    if(presentations?.positionPresentation?.slabThickness && presentations.positionPresentation.viewportType === 'volume') {
+      const slabThickness = presentations?.positionPresentation?.slabThickness;
```
Comment (HemantPatel30) [inline extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts:186]: We already have check in if condition, we can directly access slabthickness property no need for ?.

Better, we should access property only once
let SlabThickness = presentations?.positionPresentation?.slabThickness
if(SlabThickness){
    if (slabThickness <= MINIMUM_SLAB_THICKNESS) {
        viewport.setBlendMode(0); //, actorUIDs, immediate);
        viewport.setSlabThickness(MINIMUM_SLAB_THICKNESS); //, actorUIDs);
      } else {
        viewport.setBlendMode(1); //, actorUIDs, immediate);
        viewport.setSlabThickness(slabThickness); //, actorUIDs);
      }
}

File: extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts [hunk 2]
```diff
@@ -198,14 +209,22 @@ class CornerstoneViewportService
...
       viewportType:
         !viewportType || viewportType === 'stack' ? 'stack' : 'volume',
       properties,
       initialImageIndex,
       camera,
     };
+
+    if(viewportType === 'orthographic') {
+     const slabThickness:number= csViewport.getSlabThickness()? csViewport.getSlabThickness():MINIMUM_SLAB_THICKNESS;
+     presentation.slabThickness = slabThickness;
```
Comment (HemantPatel30) [inline extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts:224]: we can directly assign property with turnery operator

**Review by HemantPatel30 — CHANGES_REQUESTED** [review]

File: extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts (hunk 2 above)
Comment (kushgadhvi16) [reply extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts]: Addressed in the latest commit.

File: extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts (hunk 1 above)
Comment (kushgadhvi16) [reply extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts]: Addressed in the latest commit.

**Review by HemantPatel30 — APPROVED** [review]

**Pull request merged**